knimin/handlers/pm_plate_map.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from tornado.web import authenticated
+from knimin.handlers.base import BaseHandler
+from knimin import db
+from knimin.handlers.access_decorators import set_access
+from json import dumps
+
+
+@set_access(['Admin'])
+class PMPlateMapHandler(BaseHandler):
+    @authenticated
+    def get(self):
```

Comment:
@qiyunzhu it actually goes the other way around, each handler should be for a given AJAX call, rather than overload a single handler. This creates more modularity in the code, which makes the code easy to maintain since it is clear what each function is doing